.changes/unreleased/Dependencies-20230216-093128.yaml

```
@@ -0,0 +1,6 @@
+kind: Dependencies
+body: Updates snowflake-connector-python dependency to be <4.0.0
```

Leaving out the specifics in case they change and no one remembers to update the changelog:
```suggestion
body: Update dependency range for snowflake-connector-python
```